packages/now-client/src/upload.ts

```diff
@@ -5,6 +5,19 @@ import { fetch, API_FILES } from './utils';
 import { DeploymentError } from '.';
 import deploy, { Options } from './deploy';
 
+const isClientNetworkError = (err: Error | DeploymentError) => {
+  if (err.message) {
+    // These are common network errors that may happen occasionally and we should retry if we encounter these
+    return (
+      !err.message.includes('ETIMEDOUT') ||
+      !err.message.includes('ECONNREFUSED') ||
+      !err.message.includes('socket hang up')
```

Comment:
I'm confused why you are negating each of these.

Is "socket hang up" not a network error?